src/c++/perf_analyzer/perf_utils.h

```diff
@@ -145,4 +145,49 @@ template <Distribution distribution>
 std::function<std::chrono::nanoseconds(std::mt19937&)> ScheduleDistribution(
     const double request_rate);
 
+// Calls MPI_Init API.
+void MPI_Init(int* argc, char*** argv);
```

Comment:
Stick with pascal case with all these functions as that is what is being used in project. For all these functions.

Reply:
Renamed with pascal case 👍